Composer/packages/client/src/components/AddRemoteSkillModal/SelectIntent.tsx

```diff
@@ -0,0 +1,367 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+/** @jsx jsx */
+import { jsx, css } from '@emotion/core';
+import React, { Fragment, useState, useMemo, useEffect, useCallback } from 'react';
+import formatMessage from 'format-message';
+import { DetailsList, SelectionMode, CheckboxVisibility } from 'office-ui-fabric-react/lib/DetailsList';
+import { Selection } from 'office-ui-fabric-react/lib/Selection';
+import { Separator } from 'office-ui-fabric-react/lib/Separator';
+import { Stack, StackItem } from 'office-ui-fabric-react/lib/Stack';
+import { PrimaryButton, DefaultButton } from 'office-ui-fabric-react/lib/Button';
+import { Label } from 'office-ui-fabric-react/lib/Label';
+import { LuEditor } from '@bfc/code-editor';
+import { ScrollablePane, ScrollbarVisibility } from 'office-ui-fabric-react/lib/ScrollablePane';
+import { LuFile, LuIntentSection, SDKKinds } from '@bfc/shared';
+import { useRecoilValue } from 'recoil';
+
+import TelemetryClient from '../../telemetry/TelemetryClient';
+import { selectIntentDialog, enableOrchestratorDialog } from '../../constants';
+import httpClient from '../../utils/httpUtil';
+import luWorker from '../../recoilModel/parsers/luWorker';
+import { localeState, dispatcherState } from '../../recoilModel';
+import { recognizersSelectorFamily } from '../../recoilModel/selectors/recognizers';
+
+import { Orchestractor } from './Orchestractor';
+import { getLuDiagnostics } from './helper';
+
+const detailListContainer = css`
+  width: 100%;
+  height: 300px;
+  position: relative;
+  overflow: hidden;
+  flex-grow: 1;
+  border: 1px solid E5E5E5;
+`;
+
+interface SelectIntentProps {
+  manifest: {
+    dispatchModels: {
+      intents: Array<string> | object;
+      languages: object;
+    };
+    [key: string]: any;
+  };
+  languages: string[];
+  projectId: string;
+  luFeatures: any;

```

Comment:
this is plural, I assume it's an array? any[]?

Reply:
is a Record<string, boolean>